src/android/build.gradle

```diff
@@ -0,0 +1,7 @@
+repositories {
+	maven { url "https://jitpack.io" }
+}
+
+dependencies {
+	compile 'com.github.Jojnts:analytics-android-integration-adjust:7f588be'
```

Comment:
Is it necessary to point to the specific commit?

Reply:
No, master was fine as well apparantly! 👍